
The usual gcc-3.4 compile fixes.

  - Rene Rebe <rene@rocklinux.org>

--- ./src/file-utils.c.orig	2004-05-13 21:17:57.915157080 +0200
+++ ./src/file-utils.c	2004-05-13 21:18:02.694430520 +0200
@@ -137,6 +137,7 @@
 			break;
 
 		default:
+			;
 		}
 
 		if (full_uri)
